plugins/upload-assets-to-s3/src/tasks/upload-assets-to-s3.ts

```diff
@@ -15,8 +15,8 @@ export default class UploadAssetsToS3 extends Task<typeof UploadAssetsToS3Schema
   static description = ''
 
   static defaultOptions: UploadAssetsToS3Options = {
-    accessKeyId: process.env.aws_access_hashed_assets || '',
-    secretAccessKey: process.env.aws_secret_hashed_assets || '',
+    accessKeyId: 'aws_access_hashed_assets' || '',
+    secretAccessKey: 'aws_secret_hashed_assets' || '',
```

Comment:
**nitpick:** We can remove the `|| ''` from these lines as they were there to set a default empty string is `process.env.[key]` did not exist.